src/data/i18n.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { Locale, Localized, getTranslation, getTranslations } from './i18n'
```

Comment:
This is a rename of queryHelpers.test.tsx. No code was changed (except deletions)